torch/quantization/fuse_modules.py

```diff
@@ -32,6 +33,27 @@ def fuse_conv_bn(conv, bn):
     else:
         return torch.nn.utils.fuse_conv_bn_eval(conv, bn)
 
+def fuse_linear_bn(linear, bn):
```

Comment:
There is a naming inconsistency wrt `fuse_conv_bn_eval` which has the suffix `eval`. Is there a reason for this naming divergence?